database/migrations/2024_02_03_225118_create_calendar_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('calendar', function (Blueprint $table) {
```

Comment:
Having this entity called `Calendar` is confusing, in my opinion. It was fine in WIP, but now that we're putting it out to people, let's not set a bad example.

Each entry here isn't _really_ a calendar, is it?
Each entry here is a _calendar entry_ (aka an _event_ or a _meeting_).

**Let's please rename this entity from Calendar to Meeting** (or if you don't like that, an Event or Appointment).

I believe this is a MUST-DO, because otherwise it can confuse people. Do you need to set up a CalendarCrudController like in the demo? NO. You have a MeetingCrudController, where you use the `CalendarOperation`.

Reply:
It's true, it's much more clear with Meeting 👌